### PR TITLE
Completed - javascript-data-model / Young Jae KIM

### PR DESCRIPTION
**Pull request by abodStorage**
javascript-data-model-bookShelf
javascript-data-model-orderHistory

### CODE REVIEW

File: 0050    javascript-data-model/bookshelf.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  {
+    isbn: '978-0692232699',
+    title: 'Build APIs You Won\'t Hate: Everyone and their dog wants an API, so you should probably learn how to build them',
+    author: 'Phil Sturgeon & Laura Bohill'
+  }
+];
+
+console.log('Author of 1st book: ', bookshelf[1].author);
+console.log('ISBN of 2nd book: ', bookshelf[0].isbn);
+console.log('Title of 3rd book: ', bookshelf[2].title);
```
Comment (kurtyazdizadeh) [inline 0050    javascript-data-model/bookshelf.js]: these are not pulling the right indices. Let's double check them!

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
See comments, just one small change

File: 0050    javascript-data-model/bookshelf.js (hunk 1 above)
Comment (abodStorage) [reply 0050    javascript-data-model/bookshelf.js]: I updated code and re-committed. 
javascript-data-model-bookShelf

**Review by kurtyazdizadeh — APPROVED** [review]
Approved!

**Pull request merged**